CRD-DTR/PositiveAirwayPressureDevices/R4/resources/Questionnaire-R4-PositiveAirwayPressureDevicesFaceToFace.json

```diff
@@ -217,13 +223,27 @@
         },
         {
           "linkId": "6.2",
-          "text": "Was E0601 (CPAP) tried not effective on therapeutic trial conducted above ",
-          "type": "boolean"
+          "text": "Was E0601 (CPAP) not effective on therapeutic trial conducted above?",
+          "type": "boolean",
+          "required": true,
+          "enableWhen": [
+            {
+              "question": "6",
+              "operator": "=",
+              "answerString": "a home setting"
```

Comment:
I think I commented on this previously but I should've looked more closely.  Question 6 is the group, I think this should be referring to 6.1 instead.  Also, I believe that this should be `answerCoding`, I hadn't checked the error produced by DTR when looking into this issue.

Reply:
also the question below this has the same issue, as before

Reply:
Fixed this issue, sorry I didn't catch it before I hadn't been properly running things end-to-end. Should be all set now!